src/components/MoneyReportHeader.tsx

```diff
@@ -1,94 +1,75 @@
-import lodashGet from 'lodash/get';
-import PropTypes from 'prop-types';
 import React, {useMemo} from 'react';
 import {View} from 'react-native';
-import {withOnyx} from 'react-native-onyx';
-import _ from 'underscore';
+import {OnyxEntry, withOnyx} from 'react-native-onyx';
 import GoogleMeetIcon from '@assets/images/google-meet.svg';
 import ZoomIcon from '@assets/images/zoom-icon.svg';
 import useLocalize from '@hooks/useLocalize';
 import useThemeStyles from '@hooks/useThemeStyles';
 import useWindowDimensions from '@hooks/useWindowDimensions';
-import compose from '@libs/compose';
 import * as CurrencyUtils from '@libs/CurrencyUtils';
 import * as HeaderUtils from '@libs/HeaderUtils';
 import Navigation from '@libs/Navigation/Navigation';
 import * as ReportUtils from '@libs/ReportUtils';
-import iouReportPropTypes from '@pages/iouReportPropTypes';
-import nextStepPropTypes from '@pages/nextStepPropTypes';
-import reportPropTypes from '@pages/reportPropTypes';
 import * as IOU from '@userActions/IOU';
 import * as Link from '@userActions/Link';
 import * as Session from '@userActions/Session';
 import CONST from '@src/CONST';
 import ONYXKEYS from '@src/ONYXKEYS';
 import ROUTES from '@src/ROUTES';
+import * as OnyxTypes from '@src/types/onyx';
+import DeepValueOf from '@src/types/utils/DeepValueOf';
+import IconAsset from '@src/types/utils/IconAsset';
 import Button from './Button';
 import HeaderWithBackButton from './HeaderWithBackButton';
 import MoneyReportHeaderStatusBar from './MoneyReportHeaderStatusBar';
-import participantPropTypes from './participantPropTypes';
 import SettlementButton from './SettlementButton';
-import withWindowDimensions, {windowDimensionsPropTypes} from './withWindowDimensions';
 
-const propTypes = {
-    /** The report currently being looked at */
-    report: iouReportPropTypes.isRequired,
-
-    /** The policy tied to the money request report */
-    policy: PropTypes.shape({
-        /** Name of the policy */
-        name: PropTypes.string,
+type PaymentType = DeepValueOf<typeof CONST.IOU.PAYMENT_TYPE>;
 
-        /** Type of the policy */
-        type: PropTypes.string,
+type ThreeDotsMenuItems = {
```

Comment:
I think there is no need to create a new type in this PR now since we already have similar ones that you can just import.
[one](https://github.com/Expensify/App/blob/main/src/components/HeaderWithBackButton/types.ts#L10), [two](https://github.com/Expensify/App/blob/main/src/libs/HeaderUtils.ts#L9) (I guess we should leave just one of them).
Also I'm not sure why we need `string` in icon typing, I guess just `icon?: IconAsset;` should be ok
cc @blazejkustra

Reply:
Okay I am using from one.

Reply:
@blazejkustra  do we need string in here [two](https://github.com/Expensify/App/blob/main/src/libs/HeaderUtils.ts#L9) for type Icon?